plugins/modules/identity/keycloak/keycloak_group.py

```diff
@@ -217,30 +227,25 @@ def main():
         realm=dict(default='master'),
         id=dict(type='str'),
         name=dict(type='str'),
-        attributes=dict(type='dict')
+        attributes=dict(type='dict'),
     )
 
     argument_spec.update(meta_args)
 
     module = AnsibleModule(argument_spec=argument_spec,
                            supports_check_mode=True,
-                           required_one_of=([['id', 'name']]))
+                           required_one_of=([['id', 'name'],
+                                             ['token', 'auth_realm', 'auth_username', 'auth_password']]),
+                           required_together=([['auth_realm', 'auth_username', 'auth_password']]))
```

Comment:
Sorry, I missed that before. The same changes (add the new `required_one_of` and `required_together`) need to be done to the other modules as well.

Reply:
Yes you're right, I should have seen that.. This is fixed